src/Passes/Source/Rules/Factory.cpp

```diff
@@ -632,11 +632,16 @@ namespace quantum
 
     void RuleFactory::disableStringSupport()
     {
-        removeFunctionCall("__quantum__rt__string_create");
-        removeFunctionCall("__quantum__rt__string_update_reference_count");
-        removeFunctionCall("__quantum__rt__string_update_alias_count");
-        removeFunctionCall("__quantum__rt__message");
         removeFunctionCall("__quantum__rt__fail");
+        removeFunctionCall("__quantum__rt__message");
+        removeFunctionCall("__quantum__rt__bool_to_string");
```

Comment:
There are a couple missing compared to what is in the spec. See here for the full list of string-related runtime functions: https://github.com/microsoft/qsharp-language/blob/main/Specifications/QIR/Data-Types.md#strings

Reply:
I've updated adding all functions. It feels like we should consider what our string elimination strategy should be. Regardless, we probably should emit warnings for every function removed without further processing.